src/main/webapp/dev/js/common/instructor.js

```diff
@@ -305,7 +305,7 @@ function bindStudentPhotoHoverLink(elements) {
 /**
  * Returns the HTML required to render the soft deleted course ids as an unordered list.
  */
-function generateUnorderedListHtml(softDeletedCourseIds) {
+function generateSoftDeletedCoursesUnorderedListHtml(softDeletedCourseIds) {
```

Comment:
Consider extracting a method that generates the HTML for both courses and sessions. The reason we create this coupling is that we want to show that they should be formatted in the same way (unless you decide that there is a good reason not to format them in the same way).